src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -53,7 +53,7 @@
     <PackageReference Include="System.Reflection.Metadata" Version="1.8.0" />
     <PackageReference Include="System.Runtime" Version="4.3.1" />
     <PackageReference Include="System.Runtime.InteropServices" Version="4.3.0" />
-    <PackageReference Include="XliffTasks" Version="1.0.0-beta.19252.1" PrivateAssets="all" />
+    <PackageReference Include="XliffTasks" Version="1.0.0-beta.20206.1" PrivateAssets="all" />
```

Comment:
Not sure why this was updated in the context of C++ warning removal…

Reply:
Because the PR updates the Java.Interop submodule, whose master branch uses this version of `XliffTasks` and the difference in versions broke the build of XA with:

```
 Installing System.Reflection.Emit.ILGeneration 4.3.0.
/home/grendel/devel/mono/mono-master/lib/mono/msbuild/Current/bin/NuGet.targets(128,5): error : Detected package downgrade: XliffTasks from 1.0.0-beta.20206.1 to 1.0.0-beta.19252.1. Reference the package directly from the project to select a different version.  [/home/grendel/vc/xamarin/xamarin-android-worktrees/warnings-go-away/Xamarin.Android.sln]
/home/grendel/devel/mono/mono-master/lib/mono/msbuild/Current/bin/NuGet.targets(128,5): error :  Xamarin.Android.Build.Tasks -> Java.Interop.Tools.Cecil -> Java.Interop.Localization -> XliffTasks (>= 1.0.0-beta.20206.1)  [/home/grendel/vc/xamarin/xamarin-android-worktrees/warnings-go-away/Xamarin.Android.sln]
/home/grendel/devel/mono/mono-master/lib/mono/msbuild/Current/bin/NuGet.targets(128,5): error :  Xamarin.Android.Build.Tasks -> XliffTasks (>= 1.0.0-beta.19252.1) [/home/grendel/vc/xamarin/xamarin-android-worktrees/warnings-go-away/Xamarin.Android.sln]
/home/grendel/devel/mono/mono-master/lib/mono/msbuild/Current/bin/NuGet.targets(128,5): error : Detected package downgrade: XliffTasks from 1.0.0-beta.20206.1 to 1.0.0-beta.19252.1. Reference the package directly from the project to select a different version.  [/home/grendel/vc/xamarin/xamarin-android-worktrees/warnings-go-away/Xamarin.Android.sln]
/home/grendel/devel/mono/mono-master/lib/mono/msbuild/Current/bin/NuGet.targets(128,5): error :  Xamarin.Android.Build.Tasks -> Java.Interop.Tools.JavaCallableWrappers -> Java.Interop.Localization -> XliffTasks (>= 1.0.0-beta.20206.1)  [/home/grendel/vc/xamarin/xamarin-android-worktrees/warnings-go-away/Xamarin.Android.sln]
/home/grendel/devel/mono/mono-master/lib/mono/msbuild/Current/bin/NuGet.targets(128,5): error :  Xamarin.Android.Build.Tasks -> XliffTasks (>= 1.0.0-beta.19252.1) [/home/grendel/vc/xamarin/xamarin-android-worktrees/warnings-go-away/Xamarin.Android.sln]
```